Added setup.sh for installation : Issue #129 Fixed

SanyamGarg12 opened the pull request:
Added setup.sh as an alternative of MakeFile and updated readme accordingly.

SanyamGarg12 commented:
@NachtSpyder04

NachtSpyder04 commented on README.md:
```diff
@@ -54,20 +54,39 @@ This repository includes basics to :
 ## Installation Instructions
 
 OpenCV library and other dependency needs to be installed on your system, so follow these steps of installation:
+I'll fix the Markdown formatting in your README.md file to make it clearer and more consistent.
```
??

Sounds AI-Generated
not needed in the README, remove it

NachtSpyder04 commented on README.md:
```diff
@@ -54,20 +54,39 @@ This repository includes basics to :
-* Clone SRA-VJTI's Pixel Repository on your system
+
+
+# Installation Guide
```
Redundant line, heading is already given on line 54

NachtSpyder04 commented on README.md:
```diff
@@ -54,20 +54,39 @@ This repository includes basics to :
+
+# Installation Guide
+
+## Clone SRA-VJTI's Pixel Repository
```
follow the given format for headings, I would suggest to revert the headings

NachtSpyder04 commented on README.md:
````diff
@@ -54,20 +54,39 @@ This repository includes basics to :
+```
 
-* Change terminal directory inside the cloned repository
+## Change Directory
````
same, format issue

NachtSpyder04 reviewed (changes requested):
You have contributed to an already existing README.md. Make sure to inspect how headings and other information are in which format rather than copy-pasting your work without cross-checking.

NachtSpyder04 commented on setup.sh:
```diff
@@ -0,0 +1,26 @@
+    if grep -qi "debian" /etc/os-release; then
+        echo "Detected Debian-based system"
+        sudo apt-get update -y
+        sudo apt-get install libopencv-dev libsdl2-2.0-0 libsdl2-image-dev libsdl2-dev -y
```
add a line to install 'build-essential' package (already mentioned in #129 ) for ubuntu as well as similar command for other OSs

NachtSpyder04 reviewed (changes requested):
How should I test your script? you have not mentioned it in the description

Update your description for the same (a Dockerfile will be preferable)

NachtSpyder04 commented:
Update your PR title as well; it is unclear to those who are not aware of this issue.

SanyamGarg12 commented:
Thank you for reviewing my fixes! This is my first contribution, so I wasn’t yet familiar with some of these basics. I've also updated the README while keeping the original format intact.

To test the project setup, you can create a new virtual environment and run the bash script I added (without using make install). Instructions are included for guidance.

AryanNanda17 commented:
@SanyamGarg12, your Bash script doesn't fix #129.
It has issues. Please read the issue clearly, it states to download `git` , `make`, and other essentials before running the make command.

AryanNanda17 commented:
Thank you for your effort, but this issue has already been resolved by #131. We appreciate your contribution!
